package.json

```diff
@@ -14,6 +14,7 @@
     "build:ci": "ncc build src/adapters/github/github-actions.ts -o ./",
     "build:serverless": "ncc build src/index.ts -o ./",
     "build": "tsc",
+    "postbuild": "copyfiles src/assets/images/* lib/",
```

Comment:
I hope you verified that this works because I've had issues on occasion with the pre/post scripts.

Reply:
Yes. I ran it on the local, and then also performed some tests.